### PR TITLE
Add create-bond interface docs

### LINKED ISSUE #712: 2.x docs missing create-bond interface documentation
Mentioned here:
https://bugs.launchpad.net/maas/+bug/1595996

See also:
https://github.com/CanonicalLtd/maas-docs/issues/33

### PR DESCRIPTION
**Pull request by degville**
fixes #712 and #33 

- adds new *CLI Interface Management* doc
- describes managing bond, VLAN and bridge interfaces from the CLI
- adds new doc to the menu
- updates nodes-commission doc to include bond interfaces, plus link to CLI
- 2 x images

### CODE REVIEW

File: en/manage-cli-interfaces.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+```bash
+maas $PROFILE interfaces create-bond $SYSTEM_ID name=$BOND_NAME \
+parents=$IFACE1_ID mac_address=$MAC_ADDR \ 
+parents=$IFACE2_ID bond_mode=$BOND_MODE \
+bond_downdelay=$BOND_DOWN bond_updelay=$BOND_UP mtu=$MTU
+```
+
+Use the 'parents' parameters to define which interfaces form the aggregate
+interface. The following is an example of 'create-bond' in action:
+
```
Comment (evilnick) [inline en/manage-cli-interfaces.md]: might also be worth explaining the bond_up/bond_down. Presumably these are numeric values in milliseconds?

File: en/manage-cli-interfaces.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
+Title: CLI Interface Management
+table_of_contents: True
+
+
+# CLI Interface Management
+
+This is a list of interface management tasks to perform with the MAAS CLI. See
```
Comment (evilnick) [inline en/manage-cli-interfaces.md]: surely "which can be performed"?

File: en/manage-cli-interfaces.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+```no-highlight
+Success.
+Machine-readable output follows:
+
+```
+
+!!! Note:
+    There is no machine-readable output after the successful execution of the
+    delete command.
```
Comment (evilnick) [inline en/manage-cli-interfaces.md]: No problem with the text here, but has someone filed a bug about this?

File: en/manage-cli-interfaces.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+```
+
+Use the 'parents' parameters to define which interfaces form the aggregate
+interface. The following is an example of 'create-bond' in action:
+
+```bash
+maas admin interfaces create-bond 4efwb4 name=bond0 parents=4 \
+mac_address=52:52:00:00:00:00 parents=15 bond_mode=802.3ad bond_downdelay=200 \
+bond_updelay=200 mtu=9000
+```
```
Comment (evilnick) [inline en/manage-cli-interfaces.md]: I know the bond modes are explained elsewhere - is it worth linking to that?

File: en/manage-cli-interfaces.md (hunk 2 above)
Comment (degville) [reply en/manage-cli-interfaces.md]: I'll change this, thanks. It's the same as other CLI docs; I kept it to remain consistent with the active voice used throughout.

File: en/manage-cli-interfaces.md (hunk 1 above)
Comment (degville) [reply en/manage-cli-interfaces.md]: Will do.

File: en/manage-cli-interfaces.md (hunk 4 above)
Comment (degville) [reply en/manage-cli-interfaces.md]: There is - I added the details to the web UI section and put the link just beneath here.

File: en/manage-cli-interfaces.md (hunk 3 above)
Comment (degville) [reply en/manage-cli-interfaces.md]: Will do, thanks!

**Pull request merged**